website/docs/cdktf/concepts/remote-backends.mdx

```diff
@@ -215,10 +199,8 @@ In addition to Terraform Cloud, Terraform and CDKTF support the following backen
   ```typescript
   new S3Backend(stack, {...});
   ```
-- [swift](/language/v1.2.x/settings/backends/swift)
-  ```typescript
-  new SwiftBackend(stack, {...});
-  ```
+
+->**Note**: CDK for Terraform v0.14 deprecated the artifactory, etcd, etcdv3, manta, and swift backends. Terraform removed these backends in v1.3. For migration paths from these removed backends, refer to [Upgrading to Terraform v1.3](/language/v1.3.x/upgrade-guides).
```

Comment:
@laurapacilio took the sentence you proposed and only changed "removed" to "deprecated" as we did not yet remove them. Is this fine, or did it make things unclear?

Reply:
I think this is good! Folks know what deprecated means, so this works for me :)